manifests/plugins_dir.pp

```diff
@@ -14,7 +14,7 @@
       recurse => $recurse,
       purge   => $purge,
       force   => $force,
-      require => Package['sensu'],
+      require => Package[$::sensu::package::package_name],
     }
```

Comment:
This was "leftover" from a time when I thought the sensu-enterprise package replaced the sensu package completely. In fact, you need to install sensu and sensu-enterprise. Will clean up.